utils/hono-influxdb-connector/src/main/java/org/eclipse/kuksa/honoConnector/influxdb/InfluxDBClient.java

```diff
@@ -111,8 +111,17 @@ public void process(final MessageDTO msg) {
         if (entries == null || entries.isEmpty()) {
             return;
         }
-        
-        final Point point = createPoint(System.currentTimeMillis(), msg.getDeviceID(), entries);
+
+        //check for attribute named 'time' in message and use it as timestamp for the InfluxDb
+        long timestamp;
+        Object timeObject = entries.get("time");
```

Comment:
No magic constants introduce proper constant with value title

Reply:
I added the constant with a second commit